README.md

```diff
@@ -127,6 +127,7 @@ curl --user user:pass
 | Type  | Description  |
 |---|---|
 | main_session | represent result's page query |
+| study_session | represent study page state |
```

Comment:
@kalletlak what exactly does this represent. Is it the URL filter? If so maybe good to be explicit. Also how it differs from virtual_study. I guess for one you store the URL filter, for the other you store the list of sample ids. I.e. one is static the other is dynamic.

As a side note: an alternative approach might be to extend the virtual_study in some way, cause we are already storing the filters in there if i remember correctly

Reply:
Yes, `study_session` just stores filter state. But `virtual_study` has both filter and list of sample ids.

True, we could combine them. We should discuss about it.

cc: @alisman